transforms/universal/profiler/ray/Makefile

```diff
@@ -28,9 +28,8 @@ publish:: publish-dist publish-image
 
 publish-image:: .transforms.publish-image-ray
 
-# distribution versions is the same as image version.
 set-versions:
-	$(MAKE) TOML_VERSION=$(DOCKER_IMAGE_VERSION) .defaults.update-toml
+	$(MAKE) TRANSFORM_PYTHON_VERSION=$(DOCKER_IMAGE_VERSION) .transforms.set-versions
```

Comment:
I do not think that's working , `TRANSFORM_PYTHON_VERSION` here is the value of `PROFILER_RAY_VERSION`  while it should be `PROFILER_PYTHON_VERSION`. I tried to fix that in https://github.com/IBM/data-prep-kit/pull/323

Reply:
You are correct.  To be more clear i set to `...=not-used` when there is a no python versions (profiler, e/fdedup, doc_id).  code_quality, malware and proglang_select did need the change.

HOWEVER, in the end since all version numbers are now the same, it was working ok :)